package.json

```diff
@@ -7,8 +7,7 @@
     "url": "git@github.com:react-navigation/react-navigation.git",
```

Comment:
Can you revert the changes here?

Reply:
@satya164 - done. Note that these extra changes were from `master` though.